components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <igl/adjacency_list.h>
+#include <igl/edges.h>
+#include <igl/triangle/triangulate.h>
+#include <wmtk/Scheduler.hpp>
+#include <wmtk/TriMesh.hpp>
+
+namespace wmtk::components::internal {
+
+class EmbeddedRemeshing2D
+{
+public:
+    EmbeddedRemeshing2D(
+        Eigen::MatrixXi& E_,
+        Eigen::MatrixXd& V_,
+        double blank_rate_ = 0.5,
+        double resolute_area = 0.1);
+    void compute_bounding_vaule(double& max_x, double& max_y, double& min_x, double& min_y);
+    void process();
+
+    // can also be int, double any type, just code as bool for now.
+    Eigen::MatrixXi E;
+    Eigen::MatrixXd V;
+    double blank_rate;
```

Comment:
Please add comments what these variables represent. At least to me it is not obvious.

Reply:
// m_blank_rate is used to define how big the boundary should be
    // if the m_blank_rate = 0.5, then the boundingbox's x and y length
    // should be respectly 0.5 larger to the range of x and y of input
    // therefore, the area of the boundingbox would be as 4 times as the
    // input's boundingbox

Reply:
// m_blank_rate is used to define how big the boundary should be
    // if the m_blank_rate = 0.5, then the bounding box's x and y length
    // should be respectively 0.5 larger than the range of x and y of input
    //Therefore, the area of the bounding box is as 4 times as the input's bounding box's.